src/gurt/tests/test_gurt_telem_consumer.c

```diff
@@ -182,6 +190,42 @@ test_timer_snapshot(void **state)
 	assert((tms3.tv_sec + tms3.tv_nsec) > 0);
 }
 
+static void
+test_gauge_stats(void **state)
+{
+	struct d_tm_stats_t	stats;
+	uint64_t		val;
+	int			rc;
+
+	rc = d_tm_get_gauge(&val, &stats, shmem_root, NULL,
+			    "gurt/tests/telem/gauge-stats");
+	assert(rc == D_TM_SUCCESS);
+
+	assert_int_equal(val, 20);
+	assert_int_equal(stats.dtm_min.min_int, 2);
+	assert_int_equal(stats.dtm_max.max_int, 20);
+	assert(stats.mean - 11.0 < STATS_EPSILON);
+	assert(stats.std_dev - 5.89379 < STATS_EPSILON);
```

Comment:
Did assert_float_equal not work?

Reply:
It was not available under all of our builds.  It worked fine locally and for the builds that had the correct version of cmocka that supported it.  I didn't want to expand the scope of this PR to upgrade cmocka at the same time.

Reply:
Ah, that's fine. Thanks for the explanation.